Documentation/example-measures/example-measure-kds.json

```diff
@@ -231,7 +531,47 @@
             "language": "text/x-fhir-query",
             "expression": "Consent?_profile=http://fhir.de/ConsentManagement/StructureDefinition/Consent"
           },
-          "id": "initial-population-identifier-4"
+          "id": "initial-population-identifier-6"
+        },
+        {
+          "code": {
+            "coding": [
+              {
+                "system": "http://terminology.hl7.org/CodeSystem/measure-population",
+                "code": "measure-population"
+              }
+            ]
+          },
+          "criteria": {
+            "language": "text/fhirpath",
+            "expression": "Consent"
+          },
+          "id": "measure-population-identifier-6"
+        },
+        {
+          "code": {
+            "coding": [
+              {
+                "system": "http://terminology.hl7.org/CodeSystem/measure-population",
+                "code": "measure-observation"
+              }
+            ]
+          },
+          "criteria": {
+            "language": "text/fhirpath",
+            "expression": "Consent.subject.reference"
```

Comment:
subject does not exist for Consent - its patient instead.
see also the Consent profile: https://simplifier.net/medizininformatikinitiative-modulconsent/mii_pr_consent_einwilligung